### PR TITLE
:bust_in_silhouette: Added ModelManager

### PR DESCRIPTION
**Pull request by Ercadio**

### CODE REVIEW

File: src/service/ModelManager.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    def get_model(self, model_id : str) -> Model:
+        if model_id in self._models:
+            return self._models[model_id]
+        model_path = _get_model_path() / f"{model_id}.h5"
+        if path.exists(model_path):
+            return keras.models.load_model(model_path)
+        raise ValueError(f"No model {model_id} is available")
+
+    def save_model(self, model_id : str, model : Model) -> None:
+        self._model[model_id] = model
```
Comment (nicholas-geofroy) [inline src/service/ModelManager.py:21]: I think this should be `self._models` otherwise we have two different dictionaries

Comment (Ercadio) [reply src/service/ModelManager.py]: Good catch

Comment (Ercadio) [reply src/service/ModelManager.py]: Good catch

File: src/service/ModelManager.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        if model_id in self._models:
+            return self._models[model_id]
+        model_path = _get_model_path() / f"{model_id}.h5"
+        if path.exists(model_path):
+            return keras.models.load_model(model_path)
+        raise ValueError(f"No model {model_id} is available")
+
+    def save_model(self, model_id : str, model : Model) -> None:
+        self._models[model_id] = model
+        breakpoint()
```
Comment (tonymeng1) [inline src/service/ModelManager.py:22]: Are breakpoints enabled by default?

File: test/testModelManager.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+    def save_a_model_and_get_it(self):
+        # Simple XOR keras
+        model = tf.keras.Sequential()
+        model.add(tf.keras.Dense(2))
+        model.train(
+            [[0, 1], [1, 0], [1, 1], [0, 0]],
+            [1, 1, 0, 0])
+        self.manager.save_model("XORtest", model)
+        self.assertEqual(self.manager.get_model("XORtest"), model)
```
Comment (tonymeng1) [inline test/testModelManager.py:24]: Should the test remove the test model after it's done?

Comment (Ercadio) [reply test/testModelManager.py:24]: we can't remove a model from the manager. Once it is loaded, it's always present

**Pull request merged**